src/client/extensionActivation.ts

```diff
@@ -129,13 +129,13 @@ async function activateLegacy(
     context.subscriptions.push(manager);
     const activationPromise = manager.activate();
 
-    // Activate data science features
+    // Activate data science features after base features.
     const dataScience = serviceManager.get<IDataScience>(IDataScience);
-    dataScience.activate().ignoreErrors();
```

Comment:
This is the bug

Reply:
Interesting, so this has been broken since the split it looks like?